[18.0][MIG] document_page_procedure: Migration to 18.0 (#610)
OCA/management-system: document_page_procedure/readme/CONFIGURE.md

To be able configure your template of procedures, you need to install
mgmtsystem_manual:

- go to Management Systems \> Configuration \> Categories
- select the Procedure category

> - customize the template

Reviewer: imo this not really clear/outdated, looking at the runboat:
either we go to mgmt system -> configuration -> settings and we need to tick (i believe by default when module is installed):
![image](https://github.com/user-attachments/assets/f893c29c-bc14-42da-a209-495e48bad84d)

or what is meant here to go to mgmt system -> documentation -> pick a Procedure template and edit it?
![image](https://github.com/user-attachments/assets/feba158f-1c81-4185-9082-ba5add1f4500)

Author: > i believe by default when module is installed

That's correct, and when you disable this field, the Document Management - Wiki - Procedures module will also be uninstalled.

> To be able configure your template of procedures, you need to install
> mgmtsystem_manual:
> 
> - go to Management Systems \> Configuration \> Categories
> - select the Procedure category
> - customize the template

Reviewer 2: ```suggestion
To configure your template of procedures:

- Go to Management Systems \> Configuration \> Categories
- Select the Procedure category
- Customize the content
```

Author: If the mgmtsystem_manual module is not installed, we need to go to Knowledge > Pages > Categories instead.
